clang-tools-extra/clangd/unittests/IncludeCleanerTests.cpp

```diff
@@ -297,7 +301,8 @@ TEST(IncludeCleaner, GenerateMissingHeaderDiags) {
   Findings.UnusedIncludes.clear();
   std::vector<clangd::Diag> Diags = issueIncludeCleanerDiagnostics(
       AST, TU.Code, Findings, MockFS(),
-      {[](llvm::StringRef Header) { return Header.ends_with("buzz.h"); }});
+      {[](llvm::StringRef Header) { return Header.ends_with("buzz.h"); }},
```

Comment:
nit: a couple of parameter hint comments (`/*IgnoreHeader=*/`, `/*AngledHeaders=*/`) would aid readability here